### PR TITLE
RPS E2E test

### PR DESCRIPTION
**Pull request by marcel-apf**
Test the OCI configuration uses the correct RPS mask.
Test the RPS services uses the correct RPS mask.
Test all node network devices have the correct mask set.
Test all node pods network devices have the correct mask set.

No Polarion test id yet, approved by  @gsr-shanks

### CODE REVIEW

File: functests/1_performance/performance.go [hunk 1]
```diff
@@ -225,6 +227,72 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 		})
 	})
 
+	Context("RPS configuration", func() {
+		It("[test_id:xxxxx][ Should have the correct RPS configuration", func() {
+			if profile.Spec.CPU != nil && profile.Spec.CPU.Reserved != nil {
```
Comment (fromanirh) [inline functests/1_performance/performance.go:232]: on general we prefer to bail out early if the function preconditions aren't met. Not a big deal though.

**Review by fromanirh — COMMENTED** [review]
the direction seems good

File: functests/1_performance/performance.go (hunk 1 above)
Comment (marcel-apf) [reply functests/1_performance/performance.go:232]: I will change, thanks

**Comment by marcel-apf** [comment]
Changes:
- Fix a nit (@fromanirh)
- Rebased on master
- Removed the test-id placeholder (@gsr-shanks )

**Comment by marcel-apf** [comment]
Minor fixes

**Review by fromanirh — APPROVED** [review]
/lgtm

**Pull request merged**